pf4j/src/main/java/org/pf4j/AbstractPluginManager.java

```diff
@@ -820,18 +819,31 @@ protected synchronized void firePluginStateEvent(PluginStateEvent event) {
     }
 
     protected PluginWrapper loadPluginFromPath(Path pluginPath) throws PluginException {
-        // test for plugin duplication
+        // Test for plugin path duplication
```

Comment:
Start with small letter or not is not a problem for me but again it's good to have a unitary approach throughout the project.